api/swagger.yaml

```
@@ -4611,9 +4611,9 @@ paths:
       parameters:
         - name: "name"
           in: "query"
-          description: "Assign the specified name to the container. Must match `/?[a-zA-Z0-9_-]+`."
+          description: "Assign the specified name to the container. Must match `[a-zA-Z0-9][a-zA-Z0-9_.-]+`."
```

```suggestion
          description: "Assign the specified name to the container. Must match `/?[a-zA-Z0-9][a-zA-Z0-9_.-]+`."
```

(don't think it's needed to include the `^..$` here, as this is only the description)